modules/vstudio/tests/cs2005/test_dotnetsdk.lua

```diff
@@ -0,0 +1,188 @@
+--
+-- tests/actions/vstudio/cs2005/test_dotnetsdk.lua
+-- Test DotnetSDK feature Visual Studio 2005+ C# project.
+-- Copyright (c) 2012-2024 Jason Perkins and the Premake project
+--
+	local p = premake
+	local suite = test.declare("vstudio_cs2005_dotnetsdk")
+	local dn2005 = p.vstudio.dotnetbase
+--
+-- Setup
+--
+
+	local wks, prj, cwd
+
+--
+-- Setup and teardown
+--
+	function suite.setup()
+		cwd = os.getcwd()
+		os.chdir(_TESTS_DIR)
```

Comment:
@tritao: Is it still needed?
Apply also to
```
function suite.teardown()
		os.chdir(cwd)
	end
```

Reply:
removed it!